app/serializers/course_serializer.rb

```diff
@@ -74,6 +74,6 @@ def copy_form_required
   end
 
   def recruitment_cycle
-    object.recruitment_cycle_year
+    object.recruitment_cycle.year
```

Comment:
Can we add a quick spec for this to `spec/serializers/course_serializer_spec.rb`?

Reply:
Done